Documentation/installation/k8s-install-aks.rst

```diff
@@ -0,0 +1,59 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _aks_install:
+
+*****************************************************
+Installation using Azure CNI Powered by Cilium in AKS
+*****************************************************
+
+This guide walks you through the installation of Cilium on AKS (Azure Kubernetes Service) via 
+the `Azure Container Network Interface (CNI) Powered by Cilium
+<https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium>`__ option.
+
+Create the cluster
+==================
+
+Create an Azure CNI Powered by Cilium AKS cluster with ``network-plugin azure`` and 
+``--network-dataplane cilium``. You can create the cluster either in ``podsubnet`` or ``overlay`` mode. 
+In both modes, traffic will be routed through the Azure Virtual Network Stack. The choice between these 
+modes depends on the specific use case and requirements of the cluster. Refer to `the related documentation <https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay#choosing-a-network-model-to-use>`__  to know more about these two modes.
+.. tabs::
```

Comment:
```suggestion
modes depends on the specific use case and requirements of the cluster. Refer to `the related documentation <https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay#choosing-a-network-model-to-use>`__  to know more about these two modes.

.. tabs::
```

The new line is required here. For non-trivial doc changes, please take a look at [the Netlify preview](https://deploy-preview-28286--docs-cilium-io.netlify.app/installation/k8s-install-aks) to validate that the rendered version corresponds to your expectations. Currently, it looks like this:

![image](https://github.com/cilium/cilium/assets/17001771/eec894e0-d36b-4765-a7df-d181c880a998)

Reply:
sorry, haven't done before. how can I check preview? the above link throws me error.

Reply:
nvm, i know where to get it. Thanks for pointing out, i will check all my changes in preview